input/pagecontent/reference.md

```diff
@@ -1070,7 +1083,7 @@ In the following, we give details and examples of assignments involving various
   * recordedDate = "2013-06-08T09:57:34.2112Z"
   ```
 
-* Assignment of an integer64 (note: this data type was introduced in FHIR v4.2.0):
+* Assignment of an integer64 (note: this data type was introduced in FHIR v4.2.0): {%include tu.html%}
```

Comment:
I think the `TU` label should go _before_ the `:` (and you've done it that way elsewhere on this page).